modules/nw-ovn-ipsec-disable.adoc

```diff
@@ -8,11 +8,6 @@
 
 As a cluster administrator, you can disable IPsec encryption only if you enabled IPsec after cluster installation.
```

Comment:
>only if you enabled IPsec after cluster installation

This is not true. Whatever mode IPsec was brought on during install time (Full or External), it can be modified post install to any mode as modes are being handled by CNO now. It can also be `DISABLED` via same oc patch command which we are decrining below.

Reply:
This was a limitation in 4.14. If IPsec was brought as a part of pre-install, it can't be disabled post-install with exception of runtime enablement which can be disabled runtime.